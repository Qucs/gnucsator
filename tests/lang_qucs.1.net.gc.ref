<Qucs Dataset 0.0.19>
<indep time 4>
  +0.
  +10.0000000000000022E-6
  +20.0000000000000044E-6
  +0.00100000000000000022
</indep>
<dep _net0.Vt time>
  -1.29999974000005202
  -1.29999974000005202
  -1.29999974000005202
  -1.29999974000005202
</dep>
<dep _net1.Vt time>
  +0.
  +0.
  +0.
  +0.
</dep>
<dep V1.Vt time>
  +1.29999974000005202
  +1.29999974000005202
  +1.29999974000005202
  +1.29999974000005202
</dep>
<dep R1.Rt time>
  +50.
  +50.
  +50.
  +50.
</dep>
<dep _net0.Vt time>
  -1.29999974000005202
  -1.29999974000005202
  -1.29999974000005202
  -1.29999974000005202
</dep>
<dep _net1.Vt time>
  +0.
  +0.
  +0.
  +0.
</dep>
<dep Pr1.Vt time>
  -1.29999974000005202
  -1.29999974000005202
  -1.29999974000005202
  -1.29999974000005202
</dep>
