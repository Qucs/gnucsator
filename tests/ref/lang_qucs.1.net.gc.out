<Qucs Dataset>
default plugins: ignored
;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
store: already installed, replacing
stashing as store:0
alarm: already installed, replacing
stashing as alarm:0
iplot: already installed, replacing
stashing as iplot:0
plot: already installed, replacing
stashing as plot:0
iprint: already installed, replacing
stashing as iprint:0
print: already installed, replacing
stashing as print:0
probe: already installed, replacing
stashing as probe:0
d: already installed, replacing
stashing as d:0
diode: already installed, replacing
stashing as diode:0
npn: already installed, replacing
stashing as npn:0
pnp: already installed, replacing
stashing as pnp:0
npn1: already installed, replacing
stashing as npn1:0
pnp1: already installed, replacing
stashing as pnp1:0
Q: already installed, replacing
stashing as Q:0
bjt: already installed, replacing
stashing as bjt:0
L: already installed, replacing
stashing as L:0
  Vdc:V1 _net1 _net0   U=1.3
  R:R1 _net0 _net1   R=50 Tc1=0.0 Tc2=0.0 Temp=26.85 Tnom=26.85
  VProbe:Pr1 _net0 _net1  
open circuit: internal node 2
Gnucap   System status
iterations: op=0, dc=0, tran=11, fourier=0, total=15
transient timesteps: accepted=4, rejected=0, total=4
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=2, subckt=0, model=0, total=2
dctran density=100.0%, ac density=100.0%

<indep time 4>
 0.
 10.E-6
 20.E-6
 0.001
</indep>
<dep _net0.Vt time>
-1.3
-1.3
-1.3
-1.3
</dep>
<dep _net1.Vt time>
 0.
 0.
 0.
 0.
</dep>
<dep V1.dev.Vt time>
 1.3
 1.3
 1.3
 1.3
</dep>
<dep R1.Rt time>
 ??
 ??
 ??
 ??
</dep>
<dep _net0.Vt time>
-1.3
-1.3
-1.3
-1.3
</dep>
<dep _net1.Vt time>
 0.
 0.
 0.
 0.
</dep>
<dep Pr1.Vt time>
-1.3
-1.3
-1.3
-1.3
</dep>
